api/src/service/domain/workflow/workflowitem_create.ts

```diff
@@ -201,6 +204,17 @@ export async function createWorkflowitem(
     return new AlreadyExists(ctx, workflowitemCreated, workflowitemCreated.workflowitem.id);
   }
 
+  logger.trace({ item: workflowitemCreated }, "Check if assignee exists");
+  const isUserExists = await repository.userExists(
```

Comment:
in the TB api, we wrap return values in a `Result.Type`, which can also be an error - in this case you can check the [implementation of `userExists`](https://github.com/openkfw/TruBudget/blob/a0c42c2b74c532242eb3e87bda2e8324f622ecad/api/src/service/domain/organization/user_query.ts#L49) to see which errors it can return and in which cases.

Because of this, you must first check that the returned value is not an error and then use it the way you did. here is an example how this should be done: https://github.com/openkfw/TruBudget/blob/a0c42c2b74c532242eb3e87bda2e8324f622ecad/api/src/service/domain/organization/user_create.ts#L84